week3/homework/src/actions/deleteTodo.js

```diff
@@ -0,0 +1,22 @@
+const readTodosFile = require('./readTodosFile');
+const saveTodos = require('./saveTodos');
+
+
+async function deleteTodo(req, res) {
+    try {
+        const todos = await readTodosFile();
+        const todo = todos.find(todo => todo.id == req.params.id);
+        if (todo == null) {
+            res.send(`There is no todo id - ${req.params.id}`);
```

Comment:
great that you caught this, but aren't you missing something? res.send does not stop the execution of a function.

Reply:
Thank you a lot for reviewing and feedback, 
I’ll check again:)